Resources/Prototypes/secret_weights.yml

```
@@ -2,7 +2,8 @@
   id: Secret
   weights:
     Nukeops: 0.20
-    Traitor: 0.60
+    Traitor: 0.55
     Zombie: 0.05
     Survival: 0.10
     Revolutionary: 0.05
+    OopsAllThieves: 0.05
```

Should be way less common. This isn't a mode you're going to want to play on repeat.
```suggestion
    OopsAllThieves: 0.01
```